core/src/models/order.rs

```diff
@@ -21,10 +23,34 @@ impl Order {
             account_id: Address::repeat_byte(0x42),
             buy_token,
             sell_token,
-            buy_amount: 1_000_000_000_000_000_000,
-            sell_amount: 1_000_000_000_000_000_000,
+            numerator: 1_000_000_000_000_000_000,
+            denominator: 1_000_000_000_000_000_000,
+            remaining: 1_000_000_000_000_000_000,
+            valid_from: 0,
+            valid_until: 0,
         }
     }
+
+    pub fn compute_buy_sell_amounts(&self) -> (u128, u128) {
```

Comment:
I like that this is a method on `Order` and not a function now!

Reply:
Agree. Less important than on the struct field, but it could be nice to also rename this to be `compute_remaining_buy_sell_amounts`